recipes/libcurand/post-link.sh

```diff
@@ -0,0 +1,3 @@
+echo "By downloading and using the ${PKG_NAME} conda package, you accept the terms"  >> $PREFIX/.messages.txt
+echo "and conditions of the NVIDIA cuQuantum Software License Agreement:"            >> $PREFIX/.messages.txt
+echo "https://docs.nvidia.com/cuda/cuquantum/license.html"                           >> $PREFIX/.messages.txt
```

Comment:
This seems to be for cuQuantum. As noted in this thread ( https://github.com/conda-forge/staged-recipes/pull/21901#discussion_r1092380792 ), we can drop this